packages/gatsby-source-nacelle/package.json

```diff
@@ -0,0 +1,63 @@
+{
+  "name": "@nacelle/gatsby-source-nacelle",
+  "version": "8.0.0",
+  "description": "A Gatsby plugin for sourcing data from Nacelle's Hail Frequency API",
+  "keywords": [
+    "gatsby",
+    "gatsby-plugin",
+    "gatsby-theme",
+    "nacelle",
+    "ecommerce",
+    "e-commerce",
+    "commerce",
+    "shopify",
+    "magento"
+  ],
+  "main": "gatsby-node.js",
+  "scripts": {
+    "precommit": "lint-staged",
+    "lint": "eslint ./"
+  },
+  "author": "Nacelle Inc. (getnacelle.com)",
+  "license": "ISC",
+  "dependencies": {
+    "@nacelle/storefront-sdk": "^1.0.3",
+    "lodash.get": "^4.4.2",
+    "lodash.set": "^4.3.2"
+  },
+  "peerDependencies": {
+    "gatsby": "^4.0.0",
+    "gatsby-source-filesystem": "^4.0.0",
+    "graphql": "^15.0.0"
+  },
+  "peerDependenciesMeta": {
+    "gatsby-source-filesystem": {
+      "optional": true
+    }
+  },
+  "devDependencies": {
+    "eslint": "^7.6.0",
+    "lint-staged": "^10.2.11"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/getnacelle/nacelle-js.git"
+  },
+  "bugs": {
+    "url": "https://github.com/getnacelle/nacelle-js/issues"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "lint-staged": {
+    "**/*.(js|ts|tsx)": [
+      "prettier --write",
+      "eslint",
+      "git add"
+    ],
+    "*{json, md}": [
+      "prettier --write"
+    ]
+  },
+  "gitHead": "3153c14f529fa9f691e7cd2e6c44cee74d1a78db"
```

Comment:
I think we should remove the `gitHead` here, as it refers to an unrelated history.

```suggestion
```

Reply:
@NWRichmond, removed!

Reply:
Thank you!